app/views/assigned/clients/_client_table.haml

```diff
@@ -1,24 +1,25 @@
 .o-page__main-content.mt-3
   .clients__list
-    - clients.each do |user_client|
-      - client = user_client.client
+    - source_clients = clients.map(&:user_client)
+    - accessor = current_user.client_view_accessor.preload_source_clients(source_clients)
+    - source_clients.each do |client|
```

Comment:
@eanders I'm not sure how this view is used. Can you suggest how I can exercise it? Maybe it's part of cohorts but those don't work for me locally

Reply:
I believe you need to assign some relationships to see data there.
<img width="1108" alt="Screenshot 2025-02-06 at 5 04 41 PM" src="https://github.com/user-attachments/assets/152c44cd-d794-4015-84f9-ee5943ff4ef3" />